### PR TITLE
Add DFK banking transactions table

### PR DESCRIPTION
**Pull request by eevui**
Following discussion and schema proposed at #23 

[CSV of the latest 100 rows from snowflake](https://github.com/MetricsDAO/harmony_dbt/files/8397881/2022-03-26.8.51pm.1.csv)

### CODE REVIEW

File: models/dfk/dfk_bank_transactions.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		tags=['dfk'],
+		cluster_by=['block_timestamp']
+	)
+}}
+
+with banking_txs as (
+    select
+        tx_hash,
+        block_timestamp,
+        native_from_address as from_address,
```
Comment (antonyip) [inline models/dfk/dfk_bank_transactions.sql]: lets try not to change address names

File: models/dfk/dfk_bank_transactions.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        from_address as from_hex_address,
+        case
+            when data like '0xa59f3e0c%' then 'Deposit'
+            when data like '0x67dfd4c9%' then 'Withdraw'
+        end as type
+    from {{ ref("txs") }} txs
```
Comment (antonyip) [inline models/dfk/dfk_bank_transactions.sql]: please split this table into 2 CTES, one for just the 
- select ... from {{ ref(txs) }} where incremental_load_filter
- logics

File: models/dfk/dfk_bank_transactions.sql [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	and {{ incremental_load_filter("block_timestamp") }}
+),
+
+token_amounts as (
+    select
+        banking_txs.tx_hash as tx_hash,
+        sum(case evm_contract_address when '0x72cb10c6bfa5624dd07ef608027e366bd690048f' then event_inputs:value end) as jewel_amount,
+        sum(case evm_contract_address when '0xa9ce83507d872c5e1273e745abcfda849daa654f' then event_inputs:value end) as xjewel_amount
+    from banking_txs
+    join {{ ref("logs") }} logs on banking_txs.tx_hash = logs.tx_hash
```
Comment (antonyip) [inline models/dfk/dfk_bank_transactions.sql]:
```suggestion
    join {{ ref("logs") }} logs 
    on banking_txs.tx_hash = logs.tx_hash
```

File: models/dfk/dfk_bank_transactions.sql [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    select
+        banking_txs.tx_hash as tx_hash,
+        block_timestamp,
+        from_address,
+        from_hex_address,
+        type,
+        jewel_amount,
+        xjewel_amount
+    from banking_txs
+    join token_amounts on banking_txs.tx_hash = token_amounts.tx_hash
```
Comment (antonyip) [inline models/dfk/dfk_bank_transactions.sql]:
```suggestion
    join token_amounts
    on banking_txs.tx_hash = token_amounts.tx_hash
```

**Comment by eevui** [comment]
@antonyip addressed the reviews, for another review

**Comment by antonyip** [comment]
https://github.com/eevui/harmony_dbt/pull/2
Made you changes here.

**Comment by eevui** [comment]
Got it, just recently resetup my editor config. I'll note that we use spaces instead.

**Review by antonyip — APPROVED** [review]
LGTM

**Pull request merged**